irony-completion.el

```diff
@@ -276,12 +259,9 @@ A candidate is composed of the following elements:
  6. Post-completion data. The text to insert followed by 0 or
     more indices. These indices work by pairs and describe ranges
     of placeholder text.
-    Example: (\"(int a, int b)\" 1 6 8 13)
- 7. The availability (CXAvailabilityKind) of the candidate." 
-  (irony--awhen (irony-completion-symbol-bounds)
-    (irony-completion--filter-candidates
-     (irony--run-task
-      (irony--candidates-task nil (car it) prefix style)))))
+    Example: (\"(int a, int b)\" 1 6 8 13)"
+  (and (irony-completion-candidates-available-p)
```

Comment:
Is this code correct?
Looks like it's not only the documentation that changed, but the implementation too, the function is no longer running a task so I'm a bit surprised.